### PR TITLE
feat: add text input unstable v1 support

### PR DESCRIPTION
**Pull request by asterwyx**
Add support for text-input-unstable-v1 which is used by chromium etc.

Log: add text input unstable v1 support

### CODE REVIEW

File: src/server/qtquick/private/wquicktextinputv1.cpp [hunk 1]
```diff
@@ -0,0 +1,575 @@
+#include "wquicktextinputv1_p.h"
```
Comment (zccrs) [inline src/server/qtquick/private/wquicktextinputv1.cpp:1]: copyright

File: src/server/qtquick/private/wquicktextinputv1.cpp [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+#include <wlr/types/wlr_seat.h>
+#define static
+#include <wlr/types/wlr_compositor.h>
+#undef static
+#include <wlr/util/box.h>
+}
+
+QW_USE_NAMESPACE
+WAYLIB_SERVER_BEGIN_NAMESPACE
+struct ws_text_input_v1 {
```
Comment (zccrs) [inline src/server/qtquick/private/wquicktextinputv1.cpp:28]: What is it?

**Review by zccrs — CHANGES_REQUESTED** [review]

File: src/server/qtquick/private/wquicktextinputv1.cpp (hunk 2 above)
Comment (asterwyx) [reply src/server/qtquick/private/wquicktextinputv1.cpp:28]: Native handle defined by those who implement the protocol, like wlr_text_input_v3.

File: src/server/qtquick/private/wquicktextinputv1.cpp (hunk 1 above)
Comment (asterwyx) [reply src/server/qtquick/private/wquicktextinputv1.cpp:4]: Done

**Pull request merged**